javav2/example_code/kinesis/src/main/java/com/example/kinesis/KinesisStreamEx.java

```diff
@@ -41,12 +31,12 @@
 public class KinesisStreamEx {
 
     /**
-     * Creates a SubscribeToShardResponseHandler using the builder, which lets you set each lifecycle callback separately
-     * instead of implementing the interface
+     * Creates a SubscribeToShardResponseHandler using the builder which lets you set each lifecycle callback separately
+     * rather than implementing the interface.
      */
 
-    // Update to include a value CONSUMER_ARN
-    private static final String CONSUMER_ARN = "arn:aws:kinesis:us-east-1:814548047983:stream/StockTradeStream/consumer/StockApp:1585146130";
+    // Update to include a value CONSUMER_ARN - replace the following value
+    private static final String CONSUMER_ARN = "arn:aws:kinesis:us-east-1:000000047983:stream/StockTradeStream/consumer/StockApp:0000046130";

```

Comment:
This looks like a real account ID. Use approved fictitious numbers from the style guide, like 111122223333.

Reply:
fixed